src/api.js

```diff
@@ -1362,9 +1364,12 @@ api.declare({
     res.once('close', accept);
   });
 
-  let result = await this.workClaimer.claim(
-    provisionerId, workerType, workerGroup, workerId, count, aborted,
-  );
+  let [result] = await Promise.all([
+    this.workClaimer.claim(
+      provisionerId, workerType, workerGroup, workerId, count, aborted,
+    ),
+    this.workerInfo.provisionerSeen(provisionerId),
```

Comment:
I think we want to monitor the provisioners and workers that are out there ready to do work, rather than the provisioners and workers for which someone has called createTask.